picard/ui/itemviews/custom_columns/manager_dialog.py

```diff
@@ -0,0 +1,723 @@
+        left_panel = QtWidgets.QWidget(self)
+        left_v = QtWidgets.QVBoxLayout(left_panel)
+        left_v.setContentsMargins(0, 0, 0, 0)
+        left_v.addWidget(self._list)
+        left_buttons = QtWidgets.QHBoxLayout()
+        left_buttons.addWidget(self._btn_add)
+        left_buttons.addWidget(self._btn_duplicate)
+        left_buttons.addWidget(self._btn_delete)
+        left_v.addLayout(left_buttons)
+
+        # Middle: editor
+        self._editor_panel = QtWidgets.QWidget(self)
+        form = QtWidgets.QFormLayout(self._editor_panel)
+
+        self._title = QtWidgets.QLineEdit(self._editor_panel)
+        self._expression = ScriptTextEdit(self._editor_panel)
+        self._expression.setPlaceholderText("%artist% - %title%")
+        self._width = QtWidgets.QSpinBox(self._editor_panel)
+        self._width.setRange(DialogConfig.MIN_WIDTH, DialogConfig.MAX_WIDTH)
+        self._width.setSpecialValueText("")
+        self._width.setValue(DialogConfig.DEFAULT_WIDTH)
+        self._width.setAlignment(QtCore.Qt.AlignmentFlag.AlignRight)
+        self._width.setMaximumWidth(100)
+        self._width.setSuffix(_(' px'))
+        self._align = QtWidgets.QComboBox(self._editor_panel)
+        for label, enum_val in get_align_options():
+            self._align.addItem(_(label), enum_val)
+        self._align.setMaximumWidth(100)
+
+        self._view_selector = ViewSelector(self._editor_panel)
+        self._form_handler = ColumnFormHandler(
+            self._title,
+            self._expression,
+            self._width,
+            self._align,
+            self._view_selector,
+        )
+
+        form.addRow(_(COLUMN_INPUT_FIELD_NAMES[ColumnIndex.TITLE]) + "*", self._title)
+        form.addRow(_(COLUMN_INPUT_FIELD_NAMES[ColumnIndex.EXPRESSION]) + "*", self._expression)
+        form.addRow(_(COLUMN_INPUT_FIELD_NAMES[ColumnIndex.WIDTH]), self._width)
+        form.addRow(_(COLUMN_INPUT_FIELD_NAMES[ColumnIndex.ALIGN]), self._align)
+        form.addRow(_("Add to views"), self._view_selector)
+
+        # Display an error message to the user
+        self._error_message_display = QtWidgets.QLabel(self._editor_panel)
+        self._error_message_display.setStyleSheet("QLabel { color : red; }")
+        self._error_message_display.setWordWrap(True)
+        self._error_message_display.setText("")
+
+        form.addRow(self._error_message_display)
+
+        # Middle pane action: Update changes to selected row (non-persistent)
+        self._btn_update = QtWidgets.QPushButton(_("Update"), self._editor_panel)
+        self._btn_update.clicked.connect(self._on_update)
+        save_row = QtWidgets.QHBoxLayout()
+        save_row.addStretch(1)
+        save_row.addWidget(self._btn_update)
+        form.addRow(save_row)
+
+        # Right: docs
+        self._docs = ScriptingDocumentationWidget(include_link=True, parent=self)
+
+        # Splitter
+        self._splitter = QtWidgets.QSplitter(self)
```

Comment:
For the `PicardDialog` widget to store the window layout, it needs the splitter to have an object name.

```suggestion
        self._splitter = QtWidgets.QSplitter(self)
        self._splitter.setObjectName('column_manager_splitter')
```